### ISSUE TITLE
Querying by search term + pagination doesn't respect default sorting

### ISSUE THREAD

**Issue by justlevine** [wp-graphql/wp-graphql#2550]
### Description

When paginating through search terms, a different result order is used when a cursor is provided. 

via [Lorenzo Castro on Slack](https://wp-graphql.slack.com/archives/C3NM1M291/p1664271614776769)

### Steps to reproduce

1. Create two posts with 'Search Term' in the title or content.
2. Run the following query
```graphql
fragment SearchFragment on RootQueryToPostConnection {
  edges {
    node {
      databaseId
      title
      content
    }
    cursor
  }
}

query searchQuery( $after: String) {
  all: posts(
    first: 6, 
    where: {search: "search"}
  ) {
    ...SearchFragment
  }
  afterFirst: posts(
    first: 6,
    where: {search: "search"},
    after: $after
  ) {
    ...SearchFragment
  }
  afterFirstSorted: posts(
    first: 6, 
    where: {
      search:"search", 
      orderby: {field:DATE, order:ASC}}, 
    after:$after
  ) {
    ...SearchFragment
  }
}
```

### Additional context

![image](https://user-images.githubusercontent.com/29322304/192542379-e49d6809-ab7f-4607-a3df-7bf3c4ed6cd6.png)

### WPGraphQL Version

1.11.1

### WordPress Version

6.0.2

### PHP Version

8.0.19

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for WPGraphQL.

- [X] Yes
- [ ] My issue is with compatibility with a specific WordPress plugin, and I have listed all my installed plugins (and version info) above.

**Comment by furai** [comment]
Could it result in repeated results in pagination? Cause that's what I'm currently seeing on my end.

**Comment by justlevine** [comment]
@furai 
> Could it result in repeated results in pagination? Cause that's what I'm currently seeing on my end.

I didn't bother investigating past the initial issue, but very possibly. If the cursor is not stabilized, then there's no reason to assume that `before`/`after` are operating on the same post order.

**Comment by justlevine** [comment]
On review, I believe this is a duplicate of #1818 . If so, lets track things there.

**Comment by justlevine** [comment]
Closing in favor of #1818
